client/signup/steps/user/index.jsx

```diff
@@ -225,7 +228,7 @@ export class UserStep extends Component {
 			} );
 		}
 
-		return headerText;
+		return translate( 'Make something great' );
```

Comment:
This title is making it through to the other flows and doesn't jive so well with the subtitle:

![image](https://user-images.githubusercontent.com/6981253/50125592-96532900-0237-11e9-8af9-06111ca4e3dc.png)

and without the subtitle, it's not completely clear what you have to do on this page:

![image](https://user-images.githubusercontent.com/6981253/50125677-e500c300-0237-11e9-96c9-6f36b7f0ce88.png)

What if we:
- Changed the button copy to "Create your account". 
- Only show these changes to the `onboarding` + `onboarding-dev` flows

Reply:
I changed it on the `onboarding` flow, but it still has the subtitle there; maybe we should only change it for `onboarding-dev`?

Reply:
yeah. we can focus on the `onboarding-dev` flow. Not a problem.